src/settingsView.script.ts

```diff
@@ -97,10 +98,34 @@ window.addEventListener('message', event => {
         actionsElement.appendChild(actionElement);
     }
   } else if (method === 'models') {
-    const { configs, showModelSelector } = params;
+    const { configs, showModelSelector } = params as ModelsMessage;
     const select = document.getElementById('models') as HTMLSelectElement;
+
+    function updateConfigErrors(config: ConfigEntry) {
+      const configErrors = document.getElementById('model-errors') as HTMLParagraphElement;
+      configErrors.textContent = '';
+      for (const error of config.errors) {
+        if (!error.message)
+          continue;
+
+        const errorParagraph = document.createElement('p');
+        errorParagraph.textContent = 'Unable to load ';
```

Comment:
Since the error location might be in a different file, what about the following layout:

```
Config loading errors:
- <foo.ts:17>
- <bar.ts:42>
```

Reply:
like this?

<img width="378" alt="Screenshot 2025-04-22 at 14 03 16" src="https://github.com/user-attachments/assets/09839f32-2416-46ea-847c-f0022b37fec9" />